evaluator/data_analysis/plot_token_distribution.py

```diff
@@ -50,7 +61,8 @@ def filter_dict(counter: dict, keys: list) -> dict:
 
 def merge_keys(*arr_keys) -> list:

```

Comment:
This function seems to be dead (not referenced by anyone else) and we should remove it.